src/bootstrap/src/core/config/config.rs

```diff
@@ -2006,7 +2008,7 @@ impl Config {
                 tests,
                 enzyme,
                 plugins,
-                ccache,
+                ccache: llvm_ccache,
```

Comment:
nit: print a deprecation warning if this option was used

Reply:
Good idea, added.